### PR TITLE
Add Answer to Easypanel 🎉

### PR DESCRIPTION
**Pull request by Supernova3339**
Another Fider clone

### CODE REVIEW

File: templates/answer/index.ts [hunk 1]
```diff
@@ -0,0 +1,32 @@
...
+      proxy: {
+        port: 80,
+        secure: true,
+      },
+      mounts:[
+        {
+          type: "volume",
+          name: "answer-data",
+          mountPath: "/data",
+
```
Comment (deiucanta) [inline templates/answer/index.ts:25]: unnecessary empty line

File: templates/answer/meta.ts [hunk 2]
```diff
@@ -0,0 +1,45 @@
...
+        default: "answer",
+      },
+      appServiceImage: {
+        type: "string",
+        title: "App Service Image",
+        default: "answerdev/answer:1.0.0",
+      },
+    },
+  },
+  logo: null,
```
Comment (deiucanta) [inline templates/answer/meta.ts:33]: logo is not using the right file name - also it's too small

File: templates/answer/meta.yaml [hunk 3]
```diff
@@ -0,0 +1,32 @@
...
+description: An open-source knowledge-based community software. You can use it quickly to build Q&A community for your products, customers, teams, and more.
+instructions: Go to /install and select sqlite database
+changeLog:
+  - date: 2022-12-21
+    description: first release
+links:
+  - label: Documentation
+    url: https://answer.dev/docs
+  - label: Github
+    url: https://github.com/answerdev/answer
```
Comment (deiucanta) [inline templates/answer/meta.yaml:11]: You can also add a link to the website: https://answer.dev/

**Comment by deiucanta** [comment]
@Supernova3339 very nice app - again! Looking forward to merge this template.

**Comment by Supernova3339** [comment]
Fixes in progress! :)

**Comment by Supernova3339** [comment]
fixed

File: templates/answer/meta.yaml (hunk 3 above)
Comment (Supernova3339) [reply templates/answer/meta.yaml:11]: fixed

**Pull request merged**